mtp_cashbook/templates/cashbook/includes/table_actions.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<div class="batch-actions">
+  <p class="print">
+    <a href="#print-dialog" class="js-Dialog">{% trans "Print these transactions" %}</a>
+  </p>
+  <p class="select-all">
+    <input type="checkbox" id="select-all-{{ location }}" class="Checkbox--right js-SelectAll" data-name="{{ field_name }}">
+    <label for="select-all-{{ location }}">{% trans "Select all" %}</label>
+  </p>
+</div>
```

Comment:
Leaving as-is works for me, just so long as we've considered our reasons for one choice or the other!